cluster/functest.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
In kubevirt/kubevirt the functests go in the hack dir. Should this go there?

Reply:
I'm also curious if bash, in the end, is going to be robust enough to do this well. I know that the operator-sdk (which this project is built using) has some ideas on how to do functional testing, maybe we should be looking into that?

Reply:
I will move it to hack dir, and take a look at the functional test in the Operator-sdk, though I believe there shouldn't be any problem using bash.